bokeh/models/ranges.py

```diff
@@ -180,12 +180,12 @@ class DataRange1d(DataRange):
     as an absolute quantity. (default: ``"percent"``)
     """)
 
-    start = Either(Null, Float, Datetime, TimeDelta, help="""
+    start = Either(Float, Datetime, TimeDelta, default=float("nan"), help="""
```

Comment:
If users want to temporarily override auto-ranging, then turn it back on again, they would have to know to set `range.start = float("nan")` ? As long as we are changing this, `Auto` seems like a better default from a UX perspective since it is more self-descriptive.